core/tests/puppeteer-acceptance-tests/spec/practice-question-submitter-tests/suggest-medium-drag-and-drop-sort-question.spec.js

```diff
@@ -0,0 +1,192 @@
+    await superAdmin.createTopic({
+      name: topicName,
+      urlFragment: 'test-topic',
+      webTitleFragment: 'Between "topic name |" and "| Oppia"',
+      description: 'describe the topic',
+      thumbnail: testConstants.images.exampleTopicThumbnailImage,
+      metaContent: 'test',
+      assignedSkills: [skill],
+      subtopics: [{
+        title: 'Brief & understandable description for subtopic',
+        urlFragment: 'test-subtopic',
+        description: 'Detailed explanation of subtopic',
+        thumbnail: testConstants.images.exampleTopicThumbnailImage,
+        assignedSkills: [skill]
+      }],
+      diagnosticTestSkills: [skill],
+      isPublished: true
+    });
+
+    const topicId = await superAdmin.getTopicIdBy({ name: topicName });
+    await superAdmin.editClassroom({ topicId });
+
+    questionSubmitter = (
+      await userFactory.createNewQuestionSubmitter('questionsubmitter'));
+  }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should suggest questions by selecting the difficulty to a lesson' +
+    ' in a topic', async function() {
+    await questionSubmitter.navigateToContributorDashboard();
+    await questionSubmitter.navigateToSubmitQuestions();
+    await questionSubmitter.expectSkillOpportunityToExist({
+      topicName, skillDescription, suggestable: true
+    });
+
+    await questionSubmitter.suggestQuestionForSkillOpportunity(
+      { topicName, skillDescription });
+    await questionSubmitter.expectQuestionDifficultyChoices(
+      {
+        difficulty: DIFFICULTY_MEDIUM,
+        rubricNotes: mediumDifficultyRubricNotes
+      },
+      {
+        difficulty: DIFFICULTY_HARD,
+        rubricNotes: hardDifficultyRubricNotes
+      });
+
+    await questionSubmitter.chooseDifficulty(DIFFICULTY_MEDIUM);
+    await questionSubmitter.expectChosenDifficultyToBe(
+      DIFFICULTY_MEDIUM);
+    await questionSubmitter.expectSkillRubricNotesToHave(
+      mediumDifficultyRubricNotes);
+    await questionSubmitter.expectQuestionTextSection();
+
+    await questionSubmitter.writeQuestionText(questionText);
+    await questionSubmitter.expectQuestionTextToBe(questionText);
+
+    await questionSubmitter.addInteraction();
+    await questionSubmitter.expectInteractionChoices();
+    await questionSubmitter.navigateToCommonInteractions();
+    await questionSubmitter.chooseInteraction(
+      INTERACTION_DRAG_AND_DROP_SORT);
+    await questionSubmitter.expectDragAndDropSortInteractionEditor();
+    await (
+      questionSubmitter.expectUnmetMinimumDragAndDropSortItemCount());
+    await questionSubmitter.expectDragAndDropSortItemCountToBe(1);
+
+    await questionSubmitter.writeDragAndDropSortItems(['3', '1', '2']);
+    await questionSubmitter.expectDragAndDropSortItemCountToBe(3);
+    await questionSubmitter.allowMultipleSortItemsInSamePosition();
+    await questionSubmitter.saveInteraction();
+    await questionSubmitter.expectResponseEditor();
+
+    await questionSubmitter.chooseResponseAssertion(
```

Comment:
Same thing, top level should be:

```
questionSubmitter.addResponse(RESPONSE_EQUAL_TO_ORDERING, correctAnswer, null)
```

where the null is for tagging it with a misconception.

Reply:
The feature related to this review thread was removed.